### PR TITLE
Added SEO and meta tags for social media

### PR DESCRIPTION
**Pull request by haxybaxy**
used https://github.com/jonasmerlin/astro-seo
added keywords, open graph data, author, description

### Summary by CodeRabbit

- New Feature: Integrated the `SEO` component from the "astro-seo" package to enhance the search engine visibility of the website. This includes the generation of meta tags and Open Graph tags for improved social media sharing.
- New Feature: Set the website title to "IE Robotics and AI Club" and provided a detailed description for better search engine optimization.
- New Feature: Added specific meta tags such as `charset`, `viewport`, `keywords`, and `author` for comprehensive SEO coverage.
- New Feature: Customized the favicon link and added Twitter-specific tags for enhanced social media presence.

### CODE REVIEW

File: fe-public/src/layouts/Layout.astro [hunk 1]
```diff
@@ -9,9 +10,39 @@ const { title } = Astro.props;
...
+		<SEO
+      title="IE Robotics and AI Club"
+      description="Discover Robotics and Artificial Intelligence at our IE University Club. Explore the latest projects, workshops, and events in this exciting field."
+	  charset="UTF-8"
+
+      openGraph={{
+        basic: {
+          title: "IE Robotics and AI Club",
+          type: "website",
+          image: "club-website/fe-public/public/Logo 2.png", //NEED TO FIX AT BETA RELEASE
```
[the remark this answers is not in the document]
Comment (haxybaxy) [reply fe-public/src/layouts/Layout.astro:22]: no need, i can just upload the file on imgur

[the remark this answers is not in the document]
Comment (velocitatem) [reply fe-public/src/layouts/Layout.astro:22]: Or maybe just hard code them into `/main/fe-public/public`

[the remark this answers is not in the document]
Comment (haxybaxy) [reply fe-public/src/layouts/Layout.astro]: imgur is working fine

**Comment by velocitatem** [comment]
image

🚀 ❤️

**Comment by haxybaxy** [comment]
MERGE THAT 😍

**Pull request merged**